### PR TITLE
DAC6-336: Create start of journey for individuals

### PR DESCRIPTION
**Pull request by cameronstalker1**

### CODE REVIEW

File: conf/views/isIndividualAddressKnown.njk [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.IsIndividualAddressKnownController.onSubmit(mode).url }}" method="post">
+
+          {{ csrf() | safe }}
+
+          {{ govukRadios({
+            classes: "govuk-radios--inline",
```
Comment (helensoniega) [inline conf/views/isIndividualAddressKnown.njk:35]: Remove inline class so radios are stacked like the prototype

File: conf/views/isIndividualPlaceOfBirthKnown.njk [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.IsIndividualPlaceOfBirthKnownController.onSubmit(mode).url }}" method="post">
+
+          {{ csrf() | safe }}
+
+          {{ govukRadios({
+            classes: "govuk-radios--inline",
```
Comment (helensoniega) [inline conf/views/isIndividualPlaceOfBirthKnown.njk:35]: Here too

File: conf/views/individualDateOfBirth.njk [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+              text: messages('individualDateOfBirth.hint')
+            },
+            errorMessage: date.error,
+            items: date.items
+          }) }}
+
+          {{ govukButton({
+            text: messages("site.continue"),
+            attributes:{"id": "submit"}
+          }) }}
```
Comment (helensoniega) [inline conf/views/individualDateOfBirth.njk:52]: Change to save and continue

File: conf/views/individualName.njk [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+              name: "secondName",
+              value: form.secondName.value,
+              errorMessage: form.secondName.error,
+              classes: "govuk-!-width-three-quarters"
+            }) }}
+
+          {{ govukButton({
+            text: messages("site.continue"),
+            attributes:{"id": "submit"}
+          }) }}
```
Comment (helensoniega) [inline conf/views/individualName.njk:63]: Change to save and continue

File: conf/views/individualPlaceOfBirth.njk [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            id: "value",
+            name: "value",
+            value: form.value.value,
+            errorMessage: form.value.error
+          }) }}
+
+          {{ govukButton({
+            text: messages("site.continue"),
+            attributes:{"id": "submit"}
+          }) }}
```
Comment (helensoniega) [inline conf/views/individualPlaceOfBirth.njk:49]: Change to save and continue

File: conf/views/isIndividualAddressKnown.njk [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+              }
+            },
+            items: radios,
+            errorMessage: form.value.error
+          }) }}
+
+          {{ govukButton({
+            text: messages("site.continue"),
+            attributes:{"id": "submit"}
+          }) }}
```
Comment (helensoniega) [inline conf/views/isIndividualAddressKnown.njk:51]: Change to save and continue

File: conf/views/isIndividualPlaceOfBirthKnown.njk [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+              }
+            },
+            items: radios,
+            errorMessage: form.value.error
+          }) }}
+
+          {{ govukButton({
+            text: messages("site.continue"),
+            attributes:{"id": "submit"}
+          }) }}
```
Comment (helensoniega) [inline conf/views/isIndividualPlaceOfBirthKnown.njk:51]: Last one

**Review by helensoniega — APPROVED** [review]

**Review by soljeggers — APPROVED** [review]

**Pull request merged**